tools/lint/lint.py

```diff
@@ -430,7 +430,7 @@ def check_parsed(repo_root, path, f):
     for reftest_node in source_file.reftest_nodes:
         href = reftest_node.attrib.get("href", "").strip(space_chars)
         parts = urlsplit(href)
-        if parts.scheme or parts.netloc:
+        if (parts.scheme or parts.netloc) and not href == "about:blank":
```

Comment:
I wonder if we should make this `parts == ("about", "", "blank", "", "")` or `parts == urlsplit("about:blank")` to deal with characters being escaped in the path (`blank`). @jgraham?

Reply:
I hope no one ever does that, but I guess the latter form is OK.

Reply:
I'll update the condition, but I'm unsure of how to test this bug.
Since `about:blank` will be treated as a relative file when I add it to the href attribute in the test file.

Reply:
@jgraham What changes should I make?

Reply:
Replace the right hand side of the and here with `parts != urlsplit("about:blank")` and add a second test with `<link rel=match href=about:bl%61nk>`.